src/explore-education-statistics-common/src/components/AccordionSection.module.scss

```diff
@@ -24,6 +17,20 @@
   }
 }
 
-.sectionContentExpanded {
-  @include accordion-section-expanded();
+.section {
+  @media print {
+    display: block !important;
+    page-break-inside: avoid;
+  }
+}
+
+.sectionContent {
```

Comment:
Do we need need this class anymore? The `govuk-accordion__section` class should mean that it's visible already.